src/oaklib/implementations/funowl/funowl_implementation.py

```diff
@@ -97,7 +97,7 @@ def label(self, curie: CURIE, lang: Optional[LANGUAGE_TAG] = None) -> str:
             else:
                 raise ValueError(f"Label must be literal, not {label}")
 
-    def entities(self, filter_obsoletes=True, owl_type=None) -> Iterable[CURIE]:
+    def entities(self, filter_obsoletes=True, owl_type=None) -> Iterable[Union[URI, CURIE]]:
```

Comment:
## Alternatives?
LinkML has [`URIorCURIE`](https://github.com/linkml/linkml-runtime/blob/01838a06bd8274bc835be8ecff0a4a44335eeb99/linkml_runtime/utils/metamodelcore.py#L100) but I don't know if it's something that should be used here.

Reply:
No The URI and CURIE definition are `oaklib` specific. So the sis correct.